### PR TITLE
[Docs] ORFS doc review

### PR DESCRIPTION
**Pull request by luarss**
According to the google [docs](https://docs.google.com/document/d/1UQ-UbyEPy9UKQAtiVg6x9kPvOg-eeLFHYf_F6QHOhh0/edit)

@vvbandeira @dralabeing @vijayank88

### CODE REVIEW

**Comment by luarss** [comment]
@dralabeing Incorporated the latest changes of Supported OSes in the landing page.

File: .gitignore [hunk 1]
```diff
@@ -89,4 +89,6 @@ perf.data.old
 docs/main
 docs/build
 GitGuide.md
-mainREADME.md
+SupportedOS.md
+index2.md
+mainREADME.md
```
Comment (vvbandeira) [inline .gitignore]:
```suggestion
```

File: docs/user/BuildWithPrebuilt.md [hunk 2]
```diff
@@ -1,19 +1,9 @@
...
-- [Yosys>=0.21](https://github.com/YosysHQ/oss-cad-suite-build/blob/master/README.md#installation)
-
-```{tip} Unfortunately KLayout maintainers do not provide Debian 10/11 compatible packages. You can follow the build-from-sources instruction (Version >=0.25) and Ubuntu 22 instructions [here](https://www.klayout.de/build.html#:~:text=Building%20KLayout%20on%20Linux%20(Version%20%3E%3D%200.25)). 
-```
-
 ## Install OpenROAD
 Download pre-built binaries with self-contained dependencies
-included from the Precision Innovations' Github releases
-[here](https://github.com/Precision-Innovations/OpenROAD/actions/workflows/github-actions-build-deb-package.yml).
+included from the Precision Innovations Github releases
```
Comment (vvbandeira) [inline docs/user/BuildWithPrebuilt.md]:
```suggestion
included from the Precision Innovations' GitHub releases
```

File: docs/index.md [hunk 3]
```diff
@@ -46,33 +46,6 @@ See these [tips](user/FAQS.md#how-do-i-get-better-search-results) to help improv
 
 ### Setup
 
-#### Supported Operating Systems
```
Comment (vvbandeira) [inline docs/index.md:49]: Was this moved to some place else? @luarss

Comment (luarss) [reply docs/index.md:49]: Yes we are using the OR list for a single source of ground truth.
Changes should be in `conf.py` Line 156

File: docs/conf.py [hunk 4]
```diff
@@ -151,15 +152,51 @@ def setup(app):
     shutil.copy('../README.md', 'mainREADME.md')
     swap_prefix('mainREADME.md', '```mermaid', '```{mermaid}\n:align: center\n')   
 
+    # Grab the reference file from OR
     url = 'https://raw.githubusercontent.com/The-OpenROAD-Project/OpenROAD/master/docs/contrib/GitGuide.md'
     get_file_from_url(url, 'contrib/GitGuide.md') 
+    
+    # Temporarily using commit number, will change once OR commit merged.
+    url = 'https://raw.githubusercontent.com/The-OpenROAD-Project/OpenROAD/3563176d00daeb613eed2ccf9442137b0ae2cad3/docs/index.md'
```
Comment (vvbandeira) [inline docs/conf.py:160]: @luarss, what is the PR we are waiting here?

Comment (luarss) [reply docs/conf.py:160]: https://github.com/The-OpenROAD-Project/OpenROAD/pull/4159

Comment (luarss) [reply docs/conf.py:160]: @vvbandeira This can be changed after the companion OR PR is merged

**Comment by vvbandeira** [comment]
@luarss 
New merge conflicts

**Review by vvbandeira — APPROVED** [review]

File: docs/user/BuildWithPrebuilt.md [hunk 5]
```diff
@@ -34,7 +34,15 @@ Step 3: Run the install command based on platform use package installer.
 sudo apt install ./openroad_2.0_amd64-ubuntu20.04.deb
 ```
 
-You can install these binaries within docker as well.
+## Install Klayout and Yosys
+Please ensure the Klayout version (denoted with `klayoutVersion` variable) is consistent with the one used in [DependencyInstaller script](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/blob/master/etc/DependencyInstaller.sh). 
+
+Instructions for installing:
+- [Klayout>=0.28.8](https://www.klayout.de/build.html)
+- [Yosys>=0.21](https://github.com/YosysHQ/oss-cad-suite-build/blob/master/README.md#installation)
```
Comment (vvbandeira) [inline docs/user/BuildWithPrebuilt.md]:
```suggestion
- [Yosys>=0.39](https://github.com/YosysHQ/oss-cad-suite-build/blob/master/README.md#installation)
```

**Pull request merged**